services/bintray/bintray.tester.js

```diff
@@ -1,20 +1,18 @@
 'use strict'
 
 const Joi = require('joi')
-const createServiceTester = require('../create-service-tester')
 const { colorScheme } = require('../test-helpers')
-const { isVPlusDottedVersionNClauses } = require('../test-validators')
-
-const t = createServiceTester()
+const { isSemver } = require('../test-validators')
 
+const t = require('../create-service-tester')()
 module.exports = t
 
 t.create('version')
   .get('/asciidoctor/maven/asciidoctorj.json')
   .expectJSONTypes(
     Joi.object().keys({
       name: 'bintray',
-      value: isVPlusDottedVersionNClauses,
+      value: isSemver,
```

Comment:
`asciidoctorj` has published versions which won't validate as SemVer e.g: `1.5.3.2`, `1.5.3.1` so this will probably break again. See https://bintray.com/asciidoctor/maven/asciidoctorj How about `isVPlusDottedVersionNClausesWithOptionalSuffix` ?